ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge.cfg

```diff
@@ -530,6 +530,13 @@ route-map RM-BGP-UNDERLAY-PEERS-OUT permit 20
    description Advertise routes received from WAN iBGP towards LAN
    match route-type internal
 !
+route-map RM-BGP-WAN-PEERING-IN permit 10
```

Comment:
The interface is only a WAN interface if `wan_carrier` is set. The same model can also be used for regular l3 peerings, so we should avoid locking it to WAN. Since we also wish to have the posibility of a per-interface prefix-list, we should use a per-interface route-map. I am not sure if we want to use interface, peer-ip or something else in the name. @gmuloc ?

Reply:
Alternatively we can decide only to support bgp for wan interfaces for now.